### PR TITLE
Feature/zset sortint mget

### PR DESCRIPTION
**Pull request by karelrooted**

### CODE REVIEW

File: docs/support-commands.md [hunk 1]
```diff
@@ -161,13 +162,14 @@
...
-| Command      | Supported OR Not | Desc |
-| ------------ | ---------------- | ---- |
+| Command      | Supported OR Not | Desc                                      |
+| ------------ | ---------------- | ----------------------------------------- |
 | sicard       | √                | like scard                                |
 | siadd        | √                | like sadd, but member is int              |
 | sirem        | √                | like srem, but member is int              |
 | sirange      | √                | sirange key offset count cursor since_id  |
 | sirevrange   | √                | sirevrange key offset count cursor max_id |
+| simexist     | √                | simexist key member1 (member2 ...)        |
```
Comment (git-hulk) [inline docs/support-commands.md:172]: simexist => siexists?

**Comment by adulau** [comment]
Just curious, are you adding new commands which are not part of the official Redis core?

**Comment by git-hulk** [comment]
@adulau yeah,  we would add some new useful commands which not part of the Redis core.

**Comment by adulau** [comment]
Thanks for the feedback. That's super interesting.

**Pull request merged**